From 7a53214dd18a664eea96c9e9265374195d94a405 Mon Sep 17 00:00:00 2001
From: Be <be@mixxx.org>
Date: Thu, 30 Dec 2021 13:06:58 -0600
Subject: [PATCH] CMake: remove unnecessary macOS stuff

There is no reason for this to be there and it breaks vcpkg.
---
 pm_common/CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/pm_common/CMakeLists.txt b/pm_common/CMakeLists.txt
index 990b0e6..63d4fa1 100644
--- a/pm_common/CMakeLists.txt
+++ b/pm_common/CMakeLists.txt
@@ -52,19 +52,6 @@ if(PM_CHECK_ERRORS)
   target_compile_definitions(portmidi PRIVATE PM_CHECK_ERRORS)
 endif(PM_CHECK_ERRORS)
 
-
-if(APPLE)
-#  set(PM_OSX_VERSION "10.7" CACHE STRING
-#      "selects PM_OSX_SDK and macosx-version-min C flag")
-  set(PM_OSX_SDK "/Developer/SDKs/MacOSX${PM_OSX_VERSION}.sdk")
-  set(CMAKE_OSX_SYSROOT ${PM_OSC_SDK} CACHE 
-      PATH "-isysroot parameter for compiler")
-  set(CMAKE_C_FLAGS "-mmacosx-version-min=${PM_OSC_VERSION}" CACHE 
-      STRING "needed in conjunction with CMAKE_OSX_SYSROOT" FORCE)
-  # option(OSX_RPATH "control CMake policy for finding macOS dynamic libraries"
-  #        ON)  # enabled here to match CMake default. Is this correct?
-endif()
-
 macro(prepend_path RESULT PATH)
   set(${RESULT})
   foreach(FILE ${ARGN})
-- 
2.33.1

